jenkins/white_label.sh

```diff
@@ -30,6 +30,7 @@ paver install_pages > log/paver_install_pages.log
 organizations="HarvardMedGlobalAcademy MITxPRO"
 
 for organization in ${organizations}; do
+    export SELENIUM_BROWSER=firefox
```

Comment:
This says firefox but the title of the PR says chrome.  Can you clarify the discrepency?